src/Engine/ExpressionContainerInterface.php

```diff
@@ -48,4 +48,12 @@ public function addExpression($plugin_id, ContextConfig $config = NULL);
    */
   public function addExpressionObject(ExpressionInterface $expression);
 
+  /**
+   * Deletes an element indentified by the specified UUID in the container.
+   *
+   * @param string $uuid
+   *   The UUID of the element.
+   */
+  public function deleteExpression($uuid);
```

Comment:
This does not tell me whether deletion was successful?
I think we'll need a lookup by UUID also, so we can generate proper confirmation form with text etc.